mod/beacon/blockchain/service.go

```diff
@@ -109,7 +111,8 @@ func NewService[
 	],
 	logger log.Logger[any],
 	cs common.ChainSpec,
-	ee ExecutionEngine,
+	//nolint:lll // annoying formatter.
+	ee ExecutionEngine[*engineprimitives.PayloadAttributes[*engineprimitives.Withdrawal]],
```

Comment:
is there a reason we aren't propagating the generic all the way to the service definition, this feels like one of those moments where it'll bite us back later

Reply:
cause i was lazy and only got halfway thru haha

Reply:
LOL all good